runelite-client/src/main/java/net/runelite/client/RuneLiteModule.java

```diff
@@ -61,17 +61,23 @@
 
 	private final Supplier<Applet> clientLoader;
 	private final boolean developerMode;
+	private final String sessionfile;
```

Comment:
also imo these should be `File`s

Reply:
Maybe I was doing something wrong but bindConstant did not like Files

Reply:
Yeah because File is not constant